symfony/plugins/orangehrmTimePlugin/test/fixtures/testcases/EmployeeReportDataAPITestCase.yaml

```diff
@@ -69,6 +69,44 @@ GetAll:
         employeeId: 'E002'
         terminationId: null
 
+#  'get employee report data - without filters - self supervisor':
+#    userId: 3
+#    services:
+#      core.config_service: \OrangeHRM\Core\Service\ConfigService
+#      admin.user_service: \OrangeHRM\Admin\Service\UserService
+#      pim.employee_service: \OrangeHRM\Pim\Service\EmployeeService
+#      core.authorization.user_role_manager_helper: \OrangeHRM\Core\Authorization\Helper\UserRoleManagerHelper
+#      time.timesheet_service: \OrangeHRM\Time\Service\TimesheetService
+#      core.normalizer_service: \OrangeHRM\Core\Service\NormalizerService
+#    factories:
+#      core.authorization.user_role_manager: [ '\OrangeHRM\Core\Authorization\Manager\UserRoleManagerFactory', 'getNewUserRoleManager' ]
+#    query:
+#      name: 'employee'
+#      empNumber: 3
+#    data:
+#      - projectName: 'CUS - Core project 01'
+#        activityName: 'System Implementation'
+#        duration: '3.00'
+#      - projectName: 'CUS2 - Core project 02'
+#        activityName: 'Bug Fixes'
+#        duration: '1.00'
+#      - projectName: 'Orange - Core project 00'
+#        activityName: 'Requirement Gathering'
+#        duration: '1.00'
+#    meta:
+#      total: 3
+#      sum:
+#        hours: 5
+#        minutes: 0
+#        label: '5.00'
+#      employee:
+#        empNumber: 2
+#        lastName: 'Abel'
+#        firstName: 'Ashley'
+#        middleName: 'ST'
+#        employeeId: 'E002'
+#        terminationId: null
+
```

Comment:
This test case is commented since it is not finalised about the supervisor self view his own employee report. Can remove or uncomment once finalised.